src/python/pants/backend/jvm/tasks/jvm_compile/zinc/zinc_compile.py

```diff
@@ -438,6 +438,9 @@ def relative_to_exec_root(path):
         jdk_home=text_type(self._zinc.underlying_dist.home),
       )
       res = self.context.execute_process_synchronously_without_raising(req, self.name(), [WorkUnitLabel.COMPILER])
```

Comment:
Should this switch to `execute_process_synchronously` ?

Reply:
That'd probably work too.

Reply:
Trying it out

Reply:
Works. Updated.